pkg/remote_write.go

```diff
@@ -0,0 +1,28 @@
+package pkg
```

Comment:
better move this into a subdirectory, pkg is a bit too generic for a package name